packages/driver/cypress/e2e/cypress/request-logs.cy.ts

```diff
@@ -0,0 +1,443 @@
+import { expect } from 'chai'
+
+describe('Request Logs', () => {
```

Comment:
The majority of the difference between this and proxy-logging.cy.ts is whitespace changes due to removing a redundant `context`, set "ignore whitespace" in GitHub for an easier time reviewing the exact changes